templates/_linux.yaml

```diff
@@ -50,6 +53,24 @@ objects:
       vm.kubevirt.io/template.revision: "{{ lookup('env', 'REVISION') | default(1, true) }}"
       app: ${NAME}
   spec:
+    dataVolumeTemplates:
+      - apiVersion: cdi.kubevirt.io/v1alpha1
+        kind: DataVolume
+        metadata:
+          name: ${PVCNAME}
+        spec:
+          pvc:
+            accessModes:
+              - ReadWriteOnce
+            resources:
+              requests:
+                storage: 1Gi
```

Comment:
`1Gi` seems a bit conservative, we need to make sure that a template provides enough storage for the largest possible image that is used by it